mito-ai/src/utils/scrollMask.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Saga Inc.
+ * Distributed under the terms of the GNU Affero General Public License v3.0 License.
+ */
+
+/**
+ * Updates the mask for a scrollable container based on its scroll position
+ * @param container - The scrollable container element
+ */
+export const updateScrollMask = (container: HTMLElement | null): void => {
```

Comment:
This is sweet! Thanks for taking the time to add this polish. Looks pretty cool!